src/pages/california-privacy-notice/index.de.md

```diff
@@ -0,0 +1,158 @@
+---
+template: 'legal-page'
```

Comment:
Not sure how best to handle this, but we don't actually want legal language localized- maybe we leave it like this and close any legal language PRs? I guess passing around locale to non-localized pages would be a pain

Reply:
Hmm let's open up an issue to tackle this. Off the top of my head, we can filter out the pages that have `template: legal-page` to prevent these markdown pages from being generated, then they won't appear in the admin to edit in other languages. However, we'd have to handle redirects etc, because if the markdown for the locale doesn't exist then the page doesn't exist at that url. But yeah, we can close any legal language PRs in the meantime.

Reply:
Created #126 so I can tackle it next week